src/scrapers/otsar-hachayal.js

```diff
@@ -0,0 +1,222 @@
+import moment from 'moment';
+import { BaseScraperWithBrowser, LOGIN_RESULT } from './base-scraper-with-browser';
+import { waitForRedirect, waitForNavigation } from '../helpers/navigation';
+import { fillInput, clickButton, waitUntilElementFound } from '../helpers/elements-interactions';
+import { SHEKEL_CURRENCY, NORMAL_TXN_TYPE, SHEKEL_CURRENCY_SYMBOL } from '../constants';
+
+const BASE_URL = 'https://online.bankotsar.co.il';
+const DATE_FORMAT = 'DD/MM/YY';
+
+function getPossibleLoginResults() {
+  const urls = {};
+  urls[LOGIN_RESULT.SUCCESS] = [`${BASE_URL}/wps/myportal/FibiMenu/Online`];
+  urls[LOGIN_RESULT.INVALID_PASSWORD] = [`${BASE_URL}/LoginServices/login2.do`];
+  // TODO: support change password
+  /* urls[LOGIN_RESULT.CHANGE_PASSWORD] = [``]; */
+  return urls;
+}
+
+function getTransactionsUrl() {
+  return `${BASE_URL}/wps/myportal/FibiMenu/Online/OnAccountMngment/OnBalanceTrans/PrivateAccountFlow`;
+}
+
+function createLoginFields(credentials) {
+  return [
+    { selector: '#username', value: credentials.userCode },
+    { selector: '#password', value: credentials.password },
+  ];
+}
+
+function getAmountData(amountStr, hasCurrency = false) {
+  const amountStrCopy = amountStr.replace(',', '');
+  let currency = null;
+  let amount = null;
+  if (!hasCurrency) {
+    const amountStrCopy = amountStr.replace(',', '');
```

Comment:
@matanelgabsi seems redundant, already declared in [line 31](https://github.com/eshaham/israeli-bank-scrapers/pull/114/files#diff-62605ba186bf47d223d367d65bd74cb3R31)